examples/device/ethernet_ota/sdkconfig.defaults

```diff
@@ -1,6 +1,9 @@
 CONFIG_IDF_TARGET="esp32"
+CONFIG_BOOTLOADER_APP_ROLLBACK_ENABLE =y
 CONFIG_PARTITION_TABLE_TWO_OTA=y
 CONFIG_ESPTOOLPY_FLASHSIZE_4MB=y
+CONFIG_ESP_TASK_WDT_PANIC=y
+CONFIG_ESP_TASK_WDT_TIMEOUT_S=20
```

Comment:
The maximum image size is known (from the partition table). The default timeout shall be set to allow flashing of one OTA partition.